engines/account/app/mailers/account/signup_mailer.rb

```diff
@@ -13,8 +13,11 @@ def confirm_email(signup)
 
     private
 
+    # TODO Find a way to manage express URLs
+    #
+    # @return [String]
     def confirm_url
-      complete_signup_url(@signup, host: 'localhost:3000', email: @signup.email, token: @signup.token)
+      "http://10.0.3.70:8000?token=#{@signup.token}"
```

Comment:
This is wrong and temporary, we'll need to generate 'public' URLs with some lib in Rails. Let's talk about this.